docs/socialaccount/providers/saml.rst

```diff
@@ -73,9 +73,9 @@ via the Django admin as well:
                         # If left empty, an attempt will be done to map the attributes using
                         # built-in defaults.
                         "attribute_mapping": {
-                            "uid": "http://schemas.auth0.com/clientID",
-                            "email_verified": "http://schemas.auth0.com/email_verified",
-                            "email": "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress",
+                            "uid": "urn:oid:1.3.6.1.4.1.5923.1.1.1.6",
```

Comment:
Why this and not e.g. `urn:oid:0.9.2342.19200300.100.1.1` ?

Reply:
Also wondering, shouldn't (whichever is the right value) be added to `default_attribute_mapping` over at `saml/provider.py`?

Reply:
Good points both, I'll make the changes. I'm new to SAML integrations as of this week, so my fresh eyes are both a good and a bad thing!